server/accounts.go

```diff
@@ -3008,6 +3073,17 @@ func (dr *DirAccResolver) Start(s *Server) error {
 			}
 		}); err != nil {
 		return fmt.Errorf("error setting up pack request handling: %v", err)
+	} else if _, err = s.sysSubscribe(accListReqSubj,
```

Comment:
again, those `else` here make each line touch each other and very difficult to read.

Reply:
agreed